generator/src/main/java/com/linkedin/pegasus/generator/DataSchemaParser.java

```diff
@@ -54,38 +56,57 @@
 
   /**
    * @param resolverPath provides the search paths separated by the system file separator, or null for no search paths.
+   * @deprecated Use {@link Builder} to construct the parser.
```

Comment:
Top Level comment (line 51 ish) is probably needed to guide users to use Builder pattern since all public constructors are deprecated